spec/avram/query_logging_spec.cr

```diff
@@ -13,7 +13,7 @@ describe "Query logging" do
   it "does not log truncate statements" do
     Avram::QueryLog.dexter.temp_config do |log_io|
       TestDatabase.truncate
-      log_io.to_s.should eq("")
+      log_io.to_s.should_not contain("TRUNCATE TABLE")
```

Comment:
Since we're not truncating the db before the spec, the DatabaseInfo is not loaded/memoized and it will be fetched on this truncate call and show up in the logs https://github.com/luckyframework/avram/blob/master/src/avram/database/database_info.cr